### DISCUSSION TITLE
Welcome to xc7k325t-blinky-nextpnr Discussions!

### DISCUSSION THREAD

**Discussion by hansfbaier** [kintex-chatter/xc7k325t-blinky-nextpnr#31] [Announcements]
🎉🎉🎉🎉🎉🎉🎉🎉🎉
First signs of life from the high speed banks!
https://twitter.com/hansfbaier/status/1506927142201819136

https://github.com/f4pga/prjxray/pull/1867
🎉🎉🎉🎉🎉🎉🎉🎉🎉
@unbtorsten @jrrk2

**Comment by kazkojima** [comment]
Hi,

I'm trying to adjust nextpnr-xilinx for iob18 https://github.com/kazkojima/nextpnr-xilinx/tree/iob18-trial,  though it's still in the early stages.

Regards,
    kaz
